### ISSUE TITLE
Problem with winery installation

### ISSUE THREAD

**Issue by shashi-rks** [OpenTOSCA/winery#90]
This may not be the right place to raise this issue but I don't know any other platform. I recently installed winery with steps mentioned in InstallationGuide.pdf from OPenTOSCA website. Using Tomcat server, it was installed and I can see page http://x.x.x.x:8080/winery/servicetemplates/. But as I was trying to add artifact type, it always gave me error "Could not add Artifact Type". I tried using various namespaces as mentioned in InstallationGuide.pdf file but it failed every time. I tried to search further documentation but its not available. Can I get any help here? Has anyone seen this issue before?

**Comment by koppor** [comment]
Please use the latest Docker build as described at https://github.com/OpenTOSCA/winery#running-via-docker.

We currently have the latest build available at http://builds.opentosca.org/winery/master/winery.war, http://builds.opentosca.org/winery/master/winery-topologymodeler.war, and http://builds.opentosca.org/winery/master/topologymodeler-ui.war.

We will remove/replace the outdated installation guide soon.

**Comment by shashi-rks** [comment]
Many Thanks for the response. As I replaced wars shared by you, I no longer was able to see winery main page, instead I see following messages

"This is Winery's API. Please open the Angular UI. In development mode, it is available on http://localhost:4200/ (ng serve) or http://localhost:8080/ (configured in IntelliJ).
Configuration of IntelliJ is described here.
Swagger API description is available at http://localhost:8080/winery/swagger. Open the Swagger Petsore. Key in http://localhost:8080/winery/swagger as URL at the top of the page."

I am working within VM image and I don't have angular, eclipse installed. I am  trying to install and run winery, not build and run it using GIT code.

**Comment by koppor** [comment]
Which URL did you open? Did you click the link http://localhost:8080 (as written in the text or stated at https://github.com/OpenTOSCA/opentosca-docker#dockerized-opentosca-environment)?

**Comment by shashi-rks** [comment]
I replaced wars shared by you and tried to open the url  http://localhost:8080/winery/. Earlier it used to open winery service template page for me. Now with your WARs, it displayed a long message stated above.

I am using a ubuntu VM. I am not using docker.

**Comment by koppor** [comment]
Additionally download http://builds.opentosca.org/winery/master/winery-ui.war and deploy it as `ROOT.war`.

**Comment by shashi-rks** [comment]
Thanks for your support. I deployed WAR file shared by you and now I can see the WINERY UI. But still I was not able to add new artifacts. From Tomcat logs I found out the reason for it as "ERROR o.e.w.r.b.f.FilebasedRepository:196 createDefaultRepositoryPath - Could not create directory
java.io.IOException: Unable to create directory /opt/tomcat/winery-repository"
I manually created directory /opt/tomcat/winery-repository and now I am able to create new artifacts.

Thanks again.
